src/DataFramesMeta.jl

```diff
@@ -112,11 +112,15 @@ function fun_to_vec(kw::Expr; nolhs = false)
                 ($(Expr(:tuple, values(membernames)...)) -> $body)
             end
          else
-            output = kw.args[1]
+            if onearg(kw.args[1], :cols)
+                output = kw.args[1].args[2]
```

Comment:
is `QuoteNode` not needed here?

Reply:
We wrap in `QuoteNode` so that when it gets escaped it is treated as a Symbol. 

When a `Symbol` *not* wrapped in `QuotNode` is escaped, it looks for the variable in the current environment, which is what we want because the DataFrames expression is evaluated normally.